LICENSE.txt

```diff
@@ -200,3 +200,824 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Thanks a bunch @daisy-ycguo for adding such a comprehensive list of dependencies, you have got most of all except one `incubator-openwhisk-client-go`